src/backend/cuda/diff.cpp

```diff
@@ -15,8 +15,8 @@
 
 namespace cuda {
 
-template<typename T, bool isDiff2>
-static Array<T> diff(const Array<T> &in, const int dim) {
+template<typename T>
+Array<T> diff(const Array<T> &in, const int dim, const bool isDiff2) {
```

Comment:
It would have been nice if these changes were done in a separate PR.

Reply:
These are part of nvrtc commit, a single commit. You are just seeing them clubbed into single change as you are using `Changes` button on PR directly instead of checking out changes by commit.

Since I am going to move this into separate PR, I am going to mark this resolved.